packages/@lightningjs/ui-components/src/components/Slider/SliderLarge.stories.js

```diff
@@ -32,16 +33,32 @@ export default {
   }
 };
 
-export const Basic = () =>
-  class Basic extends lng.Component {
+export const Basic = () => {
+  const [{ value }, updateArgs] = useArgs();
+  return class Basic extends lng.Component {
     static _template() {
       return {
         Slider: {
-          type: Slider
+          type: Slider,
+          value: value,
+          signals: {
+            onChange: true
+          }
         }
       };
     }
+    // update arg control when value changes
+    onChange(value) {
+      updateArgs({ value });
+    }
   };
+};
+
+
+
+
+
+
```

Comment:
remove all of these extra empty lines. I'm surprised the linter didn't complain about these 🤔